independent-projects/arc/runtime/src/main/java/io/quarkus/arc/InstanceHandle.java

```diff
@@ -64,6 +64,12 @@ default InjectableBean<T> getBean() {
      */
     @Override
     default void close() {
+        // TODO until this contention is resolved in the specification...
```

Comment:
I think that we should replace this TODO with a tracking issue. WDYT?

Reply:
I don't think we have one, do we?

Reply:
Exactly. We should create one and use the link instead of this `TODO` ;-)

Reply:
Done.